fanuc_cr7ia_support/package.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_cr7ia_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Fanuc CR-7iA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc CR-7iA manipulators. This currently includes the /L model
+      only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>CR-7iA/L - Normal Range</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information in
+      the FANUC Robot CR-7iA Mechanical Unit Operator's Manual.
```

Comment:
Could you please be exact about which version of the manual you used? Fanuc documents are versioned, so I'd like this to be as specific as possible. It should be something like `B-NNNNNLL/VV` (with `NNNNN` the doc id, `LL` the language and `VV` the revision).